cgi/product_jqm_multilingual.pl

```diff
@@ -101,7 +101,12 @@ =head1 DESCRIPTION
 
 $code = normalize_code($code);
 
-if (not is_valid_code($code)) {
+if (not defined $User_id) {
+	$log->info("no user credentials", {code => $code, original_code => $original_code}) if $log->is_info();
```

Comment:
Should we return a `401 Unauthorized` HTTP status code or might that break clients?

Reply:
I'm not sure, so for V2 I'm keeping the 200 HTTP status code.

Reply:
I'm just wondering if some clients might give users a false sense of their data being saved, because they don't really check the response status, just the HTTP response code. But it's definitely a breaking change, so needs to be considered carefully.